lib/tasks/remove_acceptance_tests_services.rake

```diff
@@ -0,0 +1,15 @@
+desc "
+Remove the services created by acceptance tests
+Usage
+rake remove_acceptance_tests_services
+"
+task :remove_acceptance_tests_services => :environment do |_t, args|
+  services = Service.where("name like ?", "%Stormtroopers%")
```

Comment:
Shal we remove the ones that starts with acceptance test instead?

Reply:
That's me wanting to create storm trooper forms 😆 🤣

Reply:
I did initially do that, but then though someone in the team might create a form with the word "Acceptance" but they probably wouldn't create one with "Stormtroopers". Unless @njseeto would just to troll us . . . 😛

Reply:
actually i can be more specific: `Acceptance-test-Stormtrooper-FN`

i'll update to that

Reply:
Nah I use 'Star Destroyer' these days 😁